src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/8.sql

```diff
@@ -212,6 +226,16 @@ CREATE TYPE dbo.ReferenceSearchParamTableType_2 AS TABLE
     ReferenceResourceVersion int NULL
 )
 
+CREATE TYPE dbo.BulkReferenceSearchParamTableType_1 AS TABLE
```

Comment:
I believe you can remove the non-bulk types now (like dbo.ReferenceSearchParamTableType_2 above)

Reply:
Oops, good point. Removed the old types in a080589.